docs/related.md

```diff
@@ -44,7 +44,7 @@ tag on their plugin repositories.
 
 Tools layered on top of Helm or Tiller.
 
-- [Wheel](https://github.com/appscode/wheel) - Ajax friendly Helm Tiller Proxy using [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway)
+- [Swift](https://github.com/appscode/swift) - Ajax friendly Helm Tiller Proxy using [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway)
```

Comment:
Can you rename the link to Appscode Swift? People might be confused with [Apple's Swift](https://swift.org) project, so some extra clarity would be useful.